packages/babel-parser/src/parser/lval.ts

```diff
@@ -44,6 +44,12 @@ const unwrapParenthesizedExpression = (node: Node): Node => {
     : node;
 };
 
+export const enum ParseBindingListFlags {
+  ALLOW_EMPTY = 1 << 0,
+  ALLOW_MODIFIERS = 1 << 1,
+  IS_FUNCTION_PARAMS = 1 << 2,
+}
+
```

Comment:
Sorry, I forgot this.
It could be compiled into nasty object accesses when crossing modules, it would be nice if we had some way to inline it.
Do you have any ideas? I want to achieve this.

Reply:
We could both generate the object and inline usage in the same file, but unfortunately it's _very hard_ (impossible?) with our architecture to do cross-file inlining.